packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -133,6 +133,8 @@ if [ -f %{_sysconfdir}/rabbitmq/rabbitmq.conf ] && [ ! -f %{_sysconfdir}/rabbitm
     mv %{_sysconfdir}/rabbitmq/rabbitmq.conf %{_sysconfdir}/rabbitmq/rabbitmq-env.conf
 fi
 chmod -R o-rwx,g-w %{_localstatedir}/lib/rabbitmq/mnesia
+chgrp rabbitmq %{_sysconfdir}/rabbitmq
+
```

Comment:
yes, [that's better](https://github.com/rabbitmq/rabbitmq-server-release/pull/30/commits/5df0873777cbb08e7c99ece12c7461c41e78750f#diff-6d40e15a00ae78ad694619969f3cfffbR192)